src/applications/disability-benefits/all-claims/tests/content/summaryOfDisabilities.unit.spec.jsx

```diff
@@ -6,6 +6,10 @@ import { SummaryOfDisabilitiesDescription } from '../../content/summaryOfDisabil
 describe('summaryOfDisabilitiesDescription', () => {
   it('renders selected rated disabilities', () => {
     const formData = {
+      'view:claimType': {
+        'view:claimingIncrease': true,
+        'view:claimingNew': false,
+      },
       ratedDisabilities: [
```

Comment:
I feel like both `ratedDisabiliteis` and `newDisabilities` should be added to all these tests.